test/common/__snapshots__/graph.js.snap

```diff
@@ -184,11 +177,10 @@ exports[`graph #visitor should visit a model manager and create a dependency gra
    \`org.acme.hr@1.0.0.ChangeOfAddress\` --> \`concerto@1.0.0.Transaction\`
    \`org.acme.hr@1.0.0.ChangeOfAddress\` --> \`org.acme.hr@1.0.0.Person\`
    \`org.acme.hr@1.0.0.ChangeOfAddress\` --> \`org.acme.hr.base@1.0.0.Address\`
+   \`\`
+   \`\` --> \`org.acme.hr.base@1.0.0.GeneralCategory\`
```

Comment:
We appear to be missing a source vertex here?

Reply:
This happens because we don't register primitive types, but we still need to register it's associated decorator.